src/support/timestamp.c

```diff
@@ -334,13 +334,6 @@ __wt_time_aggregate_validate(
           "aggregate %s",
           __wt_time_aggregate_to_string(ta, time_string[0]));
 
-    if (ta->newest_stop_durable_ts != WT_TS_NONE &&
```

Comment:
Is the change backward compatible? If we downgrade, will the validation in the older version fail?

Reply:
Yes, it is backward compatible, we are just not taking 0 into the calculation. All other validation logic can work. As the `oldest_start_ts` cannot be zero if there are no timestamps, the above condition doesn't correct, due to that reason, I removed it.